lua/telescope/builtin.lua

```diff
@@ -97,24 +97,20 @@ builtin.commands = function()
       end
     },
     sorter = conf.generic_sorter(),
-    attach_mappings = function(prompt_bufnr, map)
-      local run_command = function()
-        local selection = actions.get_selected_entry(prompt_bufnr)
+    attach_mappings = function(prompt_bufnr)
+      actions.goto_file_selection_edit:replace(function()
```

Comment:
Most of the time i replace only `actions.goto_file_selection_edit` because all other mappings do not make sense. Or should i just replace `actions._goto_file_selection` so that `<C-v>`, etc does the same as `<CR>`